### ISSUE TITLE
Add a way to maximize the Element Call view

### ISSUE THREAD

**Issue by SimonBrandner** [vector-im/element-web#23610]
If in a group call add a way to maximize the Element Call view because otherwise the space panel, room-list and timeline take up too much space

**Comment by ThiloteE** [comment]
Yes! My group of fellow developers have encountered this problem too. Sometimes we programm together and share code to each other via screenshare. Trying to read this text is REALLY hard, because it is so tiny. Having a larger maximized screenshare would go a long way, because it automatically increases the fontsize.

We migrated from Skype to the Element Desktop app, when Skype shut down. The only reason we stuck with Skype for so long was it's "zoom in" feature. Not only was it convenient for coding together, it was also a great way to offer alleviation to people with eye impairments.
